backend_modules/libvirt/base/main.tf

```
@@ -25,7 +25,7 @@ locals {
     sles11sp4       = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.suse.de"}/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles11sp4.x86_64.qcow2"
     sles12sp3       = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.suse.de"}/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12sp3.x86_64.qcow2"
     sles12sp4       = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.suse.de"}/ibs/Devel:/Galaxy:/Terraform:/Images/images/sles12sp4.x86_64.qcow2"
-    sles12sp4o      = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.suse.de"}/install/SLE-12-SP4-JeOS-GM/SLES12-SP4-JeOS.x86_64-12.4-OpenStack-Cloud-GM.qcow2"
+    sles12sp4o      = "${var.use_mirror_images ? "http://${var.mirror}/install" : "http://schnell.suse.de/SLE12"}/SLE-12-SP4-JeOS-GM/SLES12-SP4-JeOS.x86_64-12.4-OpenStack-Cloud-GM.qcow2"
```

I think we need to do something like that:

```suggestion
    sles12sp4o      = "${var.use_mirror_images ? "http://${var.mirror}" : "http://schnell.suse.de"}/SLE12/install/SLE-12-SP4-JeOS-GM/SLES12-SP4-JeOS.x86_64-12.4-OpenStack-Cloud-GM.qcow2"
```

---

Basically, enabling the mirror option we want to use a mirror domain, and disabling it we use the origin domain. But better not mixing domains and URIs inside the domain.

---

Also, remember to update the URL in this other file:
https://github.com/uyuni-project/sumaform/blob/master/salt/mirror/etc/mirror-images.conf

And then also in:
https://gitlab.suse.de/galaxy/infrastructure/-/blob/master/srv/salt/minima/mirror-images.conf

And:
https://gitlab.suse.de/galaxy/infrastructure/-/blob/master/srv/salt/minima-bv/mirror-images.conf

Once done, you will need to apply states in the mirrors.

---

We don't want to keep the mirror outdate just for the sake of re-use what was mirrored.
Let's update everything.

---

What we do in our mirror is to mirror the exactly path used in the origin.
So, if you have ORIGIN/my_path/my_file we should provide it through MIRROR/my_path/my_file